### ISSUE TITLE
Fretboard Diagram: horizontal orientation cramped numbers

### ISSUE THREAD

**Issue by ShibuyaCyana** [musescore/MuseScore#25749]
### Issue type

Engraving bug (incorrect score rendering)

### Description with steps to reproduce

Add fretboard diagram, add fret number and fingering, and make it horizontal.

### Supporting files, videos and screenshots

![image](https://github.com/user-attachments/assets/79a7a20f-dc56-450a-87b8-f795b445dcea)

### What is the latest version of MuseScore Studio where this issue is present?

MuseScore Studio version (64-bit): 4.4.3-242971445, revision: github-musescore-musescore-eb6b367

### Regression

No.

### Operating system

OS: Windows 11 Version 2009 or later, Arch.: x86_64

### Additional context

Dec.5 nightly checked, behaves the same.
MS3 can't make fretboard diagram horizontal.

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by avvvvve** [comment]
Thanks for catching this!

A workaround for now is to make the fingering text smaller so they don't run into each other by going to 'Format > Style... > Text styles > Fretboard diagram fingering' and changing 'Size' to something like 4pt.

<img width="1187" alt="image" src="https://github.com/user-attachments/assets/ab0745a8-c248-4260-9f3a-c8a2f788dad8">

We might consider setting that to be the default size as the solution to this problem.

**Comment by ShibuyaCyana** [comment]
Hi:
Thanks for the reply. This fret number here still need some tweaks, at 4pt it still clashes with things on the 6th string, and offset properties are not really working (along with align I think).
![image](https://github.com/user-attachments/assets/1544b851-cb11-4d3a-beae-e7c8bc35365d)

**Comment by avvvvve** [comment]
Yup, it's a known issue that several of those text styles don't work (see #25574). It's a bit of a special case since these text elements are a component of an element rather than an element in their own right. We might solve this by adding the offset options to the Fretboard diagram style page rather than on the text style pages.
